javascript/javascript3/week1/Giphy_api.js

```diff
@@ -0,0 +1,42 @@
+
+let button = document.getElementById('mybutton');
+button.addEventListener('click', fetchGif);
+
+function fetchGif() {
+    let gifType = document.getElementById('giftype');
+    let gif = gifType.value;
+    let noOfGifs = document.getElementById('gifnum');
+    let Number = noOfGifs.value;
+    if (!gif) {
+        alert("Please enter the type of GIF");
```

Comment:
Dont use `alert` its bad practice and deprecated. Use a response rendered in HTML instead